uvicorn/config.py

```diff
@@ -446,6 +448,9 @@ def load(self) -> None:
                 ca_certs=self.ssl_ca_certs,
                 ciphers=self.ssl_ciphers,
             )
+            if self.ssl_options:
+                for each_option in self.ssl_options:
+                    self.ssl.options |= each_option
```

Comment:
This should probably be inside the `create_ssl_context` function.

Reply:
agreed moved to create_ssl_context.

Reply:
This is slightly out of date now in terms of some of the API calls, but I think it solves a similar issue: https://github.com/OpenCTI-Platform/client-python/blob/14cbac14afe76b3003baa5fff56c72af9eb17935/pycti/connector/opencti_connector_helper.py#L77